containers/Ideas/IdeaCreateDescription.js

```diff
@@ -68,7 +80,7 @@ export const IdeaCreateDescription = props => {
               disabled={!isValid}
             >
             </ButtonSubmit>
```

Comment:
Even though not related, but while you are near that in the code, this could be self-closing. :p